charts/incubator/misskey/questions.yaml

```diff
@@ -66,12 +66,6 @@ questions:
                               default: 3003
                               required: true
 # Include{advancedPortHTTP}
-                                - variable: targetPort
-                                  label: "Target Port"
-                                  description: "The internal(!) port on the container the Application runs on"
-                                  schema:
-                                    type: int
-                                    default: 3000
```

Comment:
This has to stay in questions tho. :)

(Removing target port from values.yaml is ok, because it gets calculated later on from the common library.)

Reply:
I see, I misunderstood you earlier comment about where it it auto-generated from.

Reply:
Cool, np!